packages/docs/src/pages/atomic-css.mdx

```diff
@@ -7,7 +7,7 @@ order: 99
 # Atomic CSS
 
 Compiled uses atomic CSS as the delivery mechanism for your styles.
-However when writing your CSS-in-JS you won't even notice that it does!
+However when writing [atomic CSS-in-JS](https://sebastienlorber.com/atomic-css-in-js) you won't even notice that it does!
```

Comment:
I don't think this is the most organic way to link out - because the point is you _arent_ writing atomic CSS, just your regular old CSS 😄.

Perhaps some link outs towards the bottom of the page might be best? This page exists to succinctly explain atomic CSS - linking out to another page kind of defeats the purpose.

Reply:
agree, added a resource section at the end.
(I think https://mxstbr.com/thoughts/tailwind/ is a good one to mention too)

Again feel free to close if you don't think it adds any value to this doc page.

Reply:
sure let's add that to resources as well and then get this merged